king/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export CXX=${PREFIX}/bin/g++
+
+mkdir -p $PREFIX/bin
+cp ./KINGcode.tar.gz $PREFIX/bin/.
+wget http://people.virginia.edu/~wc9c/KING/KINGcode.tar.gz
```

Comment:
the archive is already retrieved from the meta.yaml file, you do not need this line